<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnkaBrowserSettings</class>
 <widget class="QDialog" name="AnkaBrowserSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings | Anka Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/public/img/logo.png</normaloff>:/public/img/logo.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="search_engine_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Search Engine</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
      <property name="buddy">
       <cstring></cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="search_engine_combobox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Google</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/public/img/google.png</normaloff>:/public/img/google.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DuckDuckGo</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/public/img/duckduckgo.png</normaloff>:/public/img/duckduckgo.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brave Search</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/public/img/bravesearch.png</normaloff>:/public/img/bravesearch.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Artado</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/public/img/artado.png</normaloff>:/public/img/artado.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bing</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/public/img/bing.png</normaloff>:/public/img/bing.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Startpage</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/public/img/startpage.png</normaloff>:/public/img/startpage.png</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="tab_color_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Tab Color</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="select_tab_color">
      <property name="text">
       <string>Select Tab Color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="history_button">
      <property name="text">
       <string>Delete History</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_bookmarks_but">
      <property name="text">
       <string>Delete Bookmarks</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Please restart your browser so that the settings can be applied.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AnkaBrowserSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AnkaBrowserSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
